### ISSUE TITLE
Allow to use "-" as alternative for subdomain separator to simplify certificate handling

### ISSUE THREAD

**Issue by jmalinens** [repman-io/repman#374]
This can help to make an easier deployment of repman for some people.
My use case:
We don't have easy and ergonomic way to generate letsencrypt certs for many domains but we have *.devdomain.com cert readily available so we prefer to use one existing certificate

Example
https://myorganization-repo-repman.devdomain.com

**Comment by akondas** [comment]
If it could be done as a configuration parameter then I'm in favor. However, I see a problem with authoring composer. Because individual tokens must have a separate subdomain.

**Comment by jmalinens** [comment]
I understand Your concern!
This feature is meant only for self hosted version.
I think majority of self hosted versions will have only 1 organization.

I have updated security.yaml to also use `%domain_separator%` in regexes instead of hardcoded `[\.\-]` to avoid issues that some urls are allowed from both domains.
I have also updated comment in services.yaml explaining that alternative separator can be used in self-hosted cases for 1 organization installs.
